db/query.go

```diff
@@ -76,9 +79,10 @@ var QueryChannels = SGQuery{
 			"FROM `%s` "+
 			"USE INDEX ($idx) "+
 			"UNNEST OBJECT_PAIRS($sync.channels) AS op "+
-			"WHERE [op.name, LEAST($sync.sequence, op.val.seq),IFMISSING(op.val.rev,null),IFMISSING(op.val.del,null)]  BETWEEN  [$channelName, $startSeq] AND [$channelName, $endSeq] "+
-			"ORDER BY seq",
-		base.BucketQueryToken, base.BucketQueryToken),
+			"WHERE ([op.name, LEAST($sync.sequence, op.val.seq),IFMISSING(op.val.rev,null),"+
+			"IFMISSING(op.val.del,null)]  BETWEEN  [$channelName, $startSeq] AND [$channelName, $endSeq]) "+
```

Comment:
If you're going to break the first clause over two lines, I think doing it at the BETWEEN is more readable.

Reply:
This is fixed in latest revision.